types/TEntity.d.ts

```diff
@@ -426,3 +426,16 @@ type TContact = {
 };
 
 type TError = TContact;
+
+type TPrintableTableFilters = {
+  aBasisReached: boolean;
+  continued: boolean;
+  dismissed: boolean;
+  recall: boolean;
+  rule122: boolean;
+  setForTrial: boolean;
+  settled: boolean;
+  showAll: boolean;
+  statusUnassigned: boolean;
+  takenUnderAdvisement: boolean;
+};
```

Comment:
Just curious but is having one large file with the type definitions something that's a norm? This files looks like it's getting pretty long, I just don't know if that's something that's going to have an impact at all.